plugins/hls-class-plugin/src/Ide/Plugin/Class/Utils.hs

```diff
@@ -59,10 +59,10 @@ insertPragmaIfNotPresent state nfp pragma = do
     (_, fileContents) <- liftIO
         $ runAction "classplugin.insertPragmaIfNotPresent.GetFileContents" state
         $ getFileContents nfp
-    pm <- handleMaybeM "Unable to GetParsedModuleWithComments"
+    (pm, _) <- handleMaybeM "Unable to GetParsedModuleWithComments"
         $ liftIO
         $ runAction "classplugin.insertPragmaIfNotPresent.GetParsedModuleWithComments" state
-        $ use GetParsedModuleWithComments nfp
+        $ useWithStale GetParsedModuleWithComments nfp
```

Comment:
and here

Reply:
Interestingly, this seems like somewhere we _shouldn't_ do it. We really want the up-to-date module to compute an edit! But because we return the edit as part of the code lens, we have to. So the thing to do in the long run would be to implment `codeLens/resolve` so only the resolution method would need this. Worth a comment!

Also relevant for https://github.com/haskell/haskell-language-server/issues/3536